SteamList/Views/GradientView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  GradientView.swift
+//  SteamList
+//
+//  Created by Macbook on 12/01/2023.
+//
+
+import UIKit
+
+@IBDesignable
+class GradientView: UIView {
+    
+    @IBInspectable var firstColor: UIColor = UIColor.clear {
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var secondColor: UIColor = UIColor.clear {
+        didSet {
+            updateView()
+        }
+    }
+    
+    override class var layerClass: AnyClass {
+        get {
+            return CAGradientLayer.self
+        }
+    }
+    
+    func updateView() {
```

Comment:
This method is only being called to update the layer property. Better to ranme it to `updateLayer` or `updateGradientColors` etc

Reply:
done